### PR TITLE
PMM-3471 fix global status metric names in mysqld exporter

### PR DESCRIPTION
**Pull request by rnovikovP**

### CODE REVIEW

File: collector/global_variables.go [hunk 1]
```diff
@@ -214,7 +214,8 @@ func parseWsrepProviderOptions(opts string) float64 {
 }
 
 func validPrometheusName(s string) string {
```
Comment (AlekSi) [inline collector/global_variables.go]: Is it for metrics name or label name?
For metric name, use https://github.com/prometheus/common/blob/17f5ca1748182ddf24fc33a5a7caaaf790a52fcc/model/metric.go#L27
For label name, use https://github.com/prometheus/common/blob/17f5ca1748182ddf24fc33a5a7caaaf790a52fcc/model/labels.go#L86
Regexp below is bit wrong.

File: .travis.yml [hunk 2]
```diff
@@ -17,9 +17,11 @@ env:
 - MYSQL_IMAGE=mariadb:10.3
 - MYSQL_IMAGE=percona/percona-server:5.6
 - MYSQL_IMAGE=percona/percona-server:5.7
+- MYSQL_IMAGE=percona/percona-server:8.0
 - MYSQL_IMAGE=percona:5.5
 - MYSQL_IMAGE=percona:5.6
 - MYSQL_IMAGE=percona:5.7
+- MYSQL_IMAGE=percona:8.0
```
Comment (AlekSi) [inline .travis.yml]: 👍

File: collector/global_variables.go (hunk 1 above)
Comment (rnovikovP) [reply collector/global_variables.go]: This is code from  https://github.com/prometheus/mysqld_exporter we ported to `1.X` but have the same problem in `master`

Comment (AlekSi) [reply collector/global_variables.go]: Well, it does not make code in `1.X` better…

Comment (rnovikovP) [reply collector/global_variables.go]: looks like this is the same  code  we have in upstream https://github.com/prometheus/mysqld_exporter/blob/4feb99a85f8b105778725506abde51e77aaeecc1/collector/global_variables.go#L229

Comment (AlekSi) [reply collector/global_variables.go]: Does it make it right? :)
Really, I will fix it.

Comment (AlekSi) [reply collector/global_variables.go]: Ok, that code was correct – I missed the most critical character: `^`. Sorry for the noise.

**Pull request merged**